### DISCUSSION TITLE
Is there a cell_ontology_class equivalent for different DatasetGroups?

### DISCUSSION THREAD

**Discussion by ilibarra** [theislab/sfaira#45] [Q&A] [answered]
Quick question,

Is there an annotation, equivalent columns for  `cell_ontology_class` to cluster cells across DatasetGroup instances? Checking in case it is possible to pool together cell types with similar names, without the string matching steps.

e.g.1 `'Decidual Macrophages 1', 'Decidual Macrophages 2', 'Macrophages', 'Mono Macrophage'`
e.g.2. `['1_Endothelial', 'Endothelia', 'Endothelial', 'Endothelial Cells - AEA & DVR',`
etc.

Thanks!

**Comment by le-ander** [comment]
Just trying out the new github discussion feature here :)
any dataset super group across organs (see here how to define one: https://github.com/theislab/sfaira_tutorials/blob/master/tutorials/data_loaders.ipynb) should give you an adata with a column named 'cell_ontology_class'  which you can edit at will.

If possible for you, I would however wait until we have harmonised celltypes across organs (and matched to cell ontology), we have already progressed quite a bit with that.

**Comment by davidsebfischer** [comment, marked as answer]
Leaving an answer here for future reference: This is not supported right now because this makes cell type annotation updates to a data set a bit messier as these pooled labels may be updated to different new terms. Usually, there was a gene expression difference between clusters that received different names in the original annotation, this may result in more fine grained labels after initial publication when finer grained reference data sets become available.
